python-interface/src/MiniBotConfig/config.swarmbot.json

```diff
@@ -22,6 +22,10 @@
     }
   ],
   "sensors": [
-
+    {
+      "type": "colorSensor",
```

Comment:
are you talking about capitalization? the .py filename is ColorSensor.py and i named it colorSensor since the gpio motor's .py file was GpioMotor.py while the type on the json file for it was gpioMotor.

(sorry, i'm just not sure what you wanted, haha)